nixos/lib/test-driver/test-driver.py

```diff
@@ -738,10 +759,19 @@ def create_socket(path: str) -> socket.socket:
         self.monitor, _ = self.monitor_socket.accept()
         self.shell, _ = self.shell_socket.accept()
 
+        # Store last serial console lines for use
+        # of wait_for_console_text
+        self.last_lines: Queue = Queue(maxsize=1000)
```

Comment:
Initializing this queue with a finite buffer brings the risk that this might make the script block (we had that already once). When we make the buffer size infinite, the risk that it gets too large is negligible.

reference: https://docs.python.org/3/library/queue.html#queue.Queue

Reply:
Well, this is intended to store a few "screens", which are approximately 100 lines each. 1000 lines are 10 "screens", which is already quite a lot if you are waiting for something to appear, and you have to consider that `wait_for_console_text` is constantly reading lines. What value do you propose?